### PR TITLE
TELCODOCS-1237 release notes

### PR DESCRIPTION
**Pull request by rohennes**
[TELCODOCS-1237](https://issues.redhat.com//browse/TELCODOCS-1237): Release note to describe new tech preview features.

Version(s):
4.14

Issue:
https://issues.redhat.com/browse/TELCODOCS-1237

- https://file.emea.redhat.com/rohennes/TELCODOCS-1237-rn/release_notes/ocp-4-14-release-notes.html#ocp-4-14-networking-egress-service
- https://file.emea.redhat.com/rohennes/TELCODOCS-1237-rn/release_notes/ocp-4-14-release-notes.html#ocp-4-14-networking-metallb-vrf
- https://file.emea.redhat.com/rohennes/TELCODOCS-1237-rn/release_notes/ocp-4-14-release-notes.html#ocp-4-14-networking-nmstate-vrf

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -167,6 +167,33 @@ For more information about this more flexible SR-IOV network deployment during N
...
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-networking-egress-service"]
+==== Egress service resource to manage egress traffic for pods behind a load balancer (Technology Preview)
+
+With this update, you can use an `EgressService` custom resource (CR) to manage egress traffic for pods behind a load balancer service. This is available as a Technology Preview feature.
+
+You can use the `EgressService` CR to manage egress traffic in the following ways:
+
+* Assign a load balancer service's IP address as the source IP address for egress traffic for pods behind the load balancer service.
```
Comment (oribon) [inline release_notes/ocp-4-14-release-notes.adoc:177]: `Assign a` -> `Assign the`?
`for egress traffic for` -> `of egress traffic for` / `for egress traffic of` ?

**Review by oribon — COMMENTED** [review]
small questions, other than that the egress service part lgtm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -172,6 +172,33 @@ With this release, {product-title} updated to HAProxy 2.6.
...
+* Assign the load balancer service's IP address as the source IP address of egress traffic for pods behind the load balancer service.
+
+* Assign the egress traffic for pods behind a load balancer to a different network than the default node network.
+
+For more information, see xref:../networking/ovn_kubernetes_network_provider/configuring-egress-traffic-for-vrf-loadbalancer-services#configuring-egress-traffic-loadbalancer-services[Configuring an egress service].
+
+[id="ocp-4-14-networking-metallb-vrf"]
+==== Support for VRF specification in MetalLB's `BGPPeer` resource (Technology Preview)
+
+With this update, you can specify a Virtual Routing and Forwarding (VRF) instance in a `BGPPeer` custom resource. MetalLB can advertise services and make routing decisions based on the routing information in the VRF. This is available as a Technology Preview feature.
```
Comment (fedepaol) [inline release_notes/ocp-4-14-release-notes.adoc:191]:
```suggestion
With this update, you can specify a Virtual Routing and Forwarding (VRF) instance in a `BGPPeer` custom resource. MetalLB can advertise services via the interfaces belonging to the VRF. This is available as a Technology Preview feature.
```

**Review by skopacz1 — COMMENTED** [review]
Left a few nits, but otherwise looks good!

**Pull request merged**